Text/ExtractCJKTextByPatternMatch/ExtractCJKTextByPatternMatch.vcxproj.user

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup />
```

Comment:
The configs aren't present here?

Reply:
I have added the missing configs to address this